mitodl/ocw-studio
# studio-side fix for linebreak table issue

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #1074

#### What's this PR do?

This removes a line where we were doing `content.replace('\n\n', '\n')` specifically on the content nested inside of tables in our markdown table support plugin. The `.replace` call was added because it was a workaround for a different spacing issue we were having before, but **a)** it's causing weird spacing issues (see #1074) and **b)** it doesn't appear to be necessary anymore.

_**note**: we're going to have to add a small CSS fix over in ocw-hugo-themes to deal with some extraneous empty paragraphs that Hugo renders when there are `\n\n` sequences_

#### How should this be manually tested?

Read through the issue and make sure you can repro the issue. Then check out this branch and confirm that editing works smoothy.
